src/main.cpp

```diff
@@ -4364,18 +4370,19 @@ std::string GetWarnings(const std::string& strFor)
     // Misc warnings like out of disk space and clock is wrong
     if (strMiscWarning != "")
     {
-        strStatusBar = strGUI = strMiscWarning;
+        strStatusBar = strMiscWarning;
+        strGUI += strGUI.empty() ? "" : uiAlertSeperator + strMiscWarning;
```

Comment:
Jup, I meant round brackets. (Not curly) 😣

No need for an extra if statement. You can put it in two lines if one is too long:

```diff
diff --git a/src/main.cpp b/src/main.cpp
index 505fd3a..e041472 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -4373,3 +4373,4 @@ std::string GetWarnings(const std::string& strFor)
         strStatusBar = strMiscWarning;
-        strGUI += strGUI.empty() ? "" : uiAlertSeperator + strMiscWarning;
+        strGUI += (strGUI.empty() ? "" : uiAlertSeperator) +
+                  strMiscWarning;
     }